src/js/components/locations-configuration/SuccessMessage.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+
+import PropTypes from 'prop-types';
+import { withRouter } from 'react-router-dom';
+
+import Translate from 'utils/Translate';
+
+import 'components/locations-configuration/SuccessMessage.scss';
+
+
+const SuccessMessage = ({ setShowSuccessMessage, history }) => (
+  <div className="success-main">
```

Comment:
I think you should use react-modal here, so all of this custom styling will not be needed